client/src/domain/Seat/index.js

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+import './style.scss';
+
+const Seat = () => {
+  const [selected, setSelected] = useState(false);
+
+  const seatNr = 1;
+  // TO DO move the handle function to the parent component
+  const handleSeatSelected = (e) => {
+    e.target.classList.toggle('selected');
+    setSelected((prevSelected) => !prevSelected);
+    console.log(selected);
+  };
+  return (
+    <div>
```

Comment:
I think we don't really need to wrap it in a div. I think just button tag will suffice.